tests_config/run_spec.js

```diff
@@ -229,66 +220,66 @@ global.run_spec = (fixtures, parsers, options) => {
             createSnapshot(
               codeForSnapshot,
               resultForSnapshot,
-              composeOptionsForSnapshot(baseOptions, parsers),
+              composeOptionsForSnapshot(firstFormatResult.options, parsers),
               { codeOffset }
             )
           ).toMatchSnapshot();
         }
       });
 
-      const parsersToVerify = parsers.slice(1);
-      if (parsers.includes("typescript") && !parsers.includes("babel-ts")) {
-        parsersToVerify.push("babel-ts");
-      }
-
-      for (const parser of parsersToVerify) {
-        const verifyOptions = { ...baseOptions, parser };
-
+      for (const parser of verifyParsers) {
+        const verifyOptions = { ...firstFormatResult.options, parser };
         test(`verify (${parser})`, () => {
           if (
             parser === "babel-ts" &&
-            options &&
-            (options.disableBabelTS === true ||
-              (Array.isArray(options.disableBabelTS) &&
-                options.disableBabelTS.includes(name)))
+            formatOptions &&
+            (formatOptions.disableBabelTS === true ||
+              (Array.isArray(formatOptions.disableBabelTS) &&
+                formatOptions.disableBabelTS.includes(name)))
           ) {
             expect(() => {
-              format(input, filename, verifyOptions);
+              format(firstFormatResult.input, verifyOptions);
             }).toThrow(TEST_STANDALONE ? undefined : SyntaxError);
           } else {
-            const verifyOutput = format(input, filename, verifyOptions);
-            expect(visualizeEndOfLine(verifyOutput)).toEqual(visualizedOutput);
+            const verifyFormatResult = format(
+              firstFormatResult.input,
+              verifyOptions
+            );
+            expect(verifyFormatResult.eolVisualizedOutput).toEqual(
+              firstFormatResult.eolVisualizedOutput
+            );
           }
         });
       }
 
-      const isUnstable = unstableTests.get(filename);
-      const isUnstableTest = isUnstable && isUnstable(options || {});
+      const isUnstable = isUnstableTest(filename, formatOptions);
       if (
         DEEP_COMPARE &&
-        (formatted !== input || isUnstableTest) &&
-        typeof rangeStart === "undefined" &&
-        typeof rangeEnd === "undefined" &&
-        typeof cursorOffset === "undefined" &&
-        !TEST_CRLF
```

Comment:
This enables `second format test` on windows. Don't know I why I skip this before.